index.html

```diff
@@ -9,7 +9,7 @@
 		<title>mandy b. bakes</title>
 		<meta charset="utf-8" />
 		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-		<link rel="stylesheet" href="assets/css/main.css" />
+		<link rel="stylesheet" href="dist/css/main.css" />
```

Comment:
I just realized I don't actually have dist folder on my branch. How was this generated and did we need it?

Reply:
In another word, if the default version was fine using assets folder, we should stick with making changes to the assets folder and remove dist folder.

Reply:
Oh you are right, we can just keep using the assets folder.

I realized it is the default settings on my plugin to output to a dist folder, since that is what I use in projects with Next.js or React.

Reply:
removed the dist folder and reconnected the html pages to the /assets/css/main.css